client/layouts/default.vue

```diff
@@ -130,11 +127,21 @@ interface LayoutButton {
 
 export default Vue.extend({
   mixins: [settingsHelper],
+  props: {
+    keepAliveOptions: {
+      type: Object as () => Record<string, unknown>,
+      default: (): Record<string, unknown> => {
+        return {
+          max: 10,
+          exclude: ['fullscreen-playback']
```

Comment:
We should probably report the fact that this is not working upstream

Reply:
What's not working?

Reply:
The page is not being excluded from the keep-alive. According to Nuxt docs I think our approach for excluding it is right, but it doesn't work.

Reply:
I guess you could at least bring that to them yeah